packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -21,13 +143,95 @@ export default {
 
 .c-mediaElement {
     display: flex;
+    width: 100%;
+}
+.c-mediaElement--stack {
+    flex-direction: column;
+}
+.c-mediaElement--stack.c-mediaElement--reverse {
+    flex-direction: column-reverse;
+}
```

Comment:
```suggestion
.c-mediaElement--stack {
    flex-direction: column;
 
       &.c-mediaElement--reverse {
           flex-direction: column-reverse;
       }
}

```

Reply:
I'd also add code comments to the modifiers to describe what they do. E.g. from button:

```scss
/**
 * Modifier – .o-btn--secondary
 *
 * Accompanying secondary style button (knocked back slightly from the primary button)
 */
```

Reply:
Yep, I have now added these to all the modifiers